### ISSUE TITLE
Linechart inverted

### ISSUE THREAD

**Issue by HPRaval** [danielgindi/Charts#2490]
I want to create line chart with inverted axis, like this

![screen shot 2017-06-01 at 1 16 09 pm](https://cloud.githubusercontent.com/assets/24671241/26669719/87135cbc-46cc-11e7-9e1a-8312e3509c62.png)

For this i have tried following

 linechart.leftAxis.enabled = false
        linechart.xAxis.labelPosition = .bottom
        linechart.rightAxis.axisMinimum = -100

But i am not getting output as expected...my output is as below

![screen shot 2017-06-01 at 1 17 41 pm](https://cloud.githubusercontent.com/assets/24671241/26669776/b93daefe-46cc-11e7-8042-6c310ed07f42.png)

Basically i want to flip x and y axis.

**Comment by liuxuan30** [comment]
try
```swift
    /// flag that indicates if the axis is inverted or not
    open var inverted = false
```

**Comment by HPRaval** [comment]
Thnx for your time and suggestion but that inverts values of left / right axis. Its like ascending-desending the values of y-axis. But i need to invert axis. Means x axis to y-axis. Are you getting it?

**Comment by liuxuan30** [comment]
OK I see. y axis and x axis are flexible enough to switch in your case. As long as you know the (x, y) then you can draw your chart and format your x/y axis labels. It should be done by you. Meaning you write code to make the library's y axis looks like your x axis.

**Comment by thincamel** [comment]
please see the result by changed chart.inverted
https://code.hcharts.cn/highcharts/hhhhy1
